ups/eupspkg.cfg.sh

```diff
@@ -0,0 +1,2 @@
+NJOBS=1
+export SCONSFLAGS=$SCONSFLAGS" USE_UNKNOWN_VARS=true PREFIX="$PREFIX"/lib"
```

Comment:
It's unusual to have a PREFIX point to a lib directory.  You should at least have a comment to explain this.